src/include/units/quantity_kind.h

```diff
@@ -104,6 +104,9 @@ class quantity_kind {
   quantity_kind& operator=(const quantity_kind&) = default;
   quantity_kind& operator=(quantity_kind&&) = default;
 
+  template <PointOrigin Orig>
+  [[nodiscard]] constexpr quantity_point_kind<downcast_point_kind<kind_type>, unit, rep, Orig> absolute() const noexcept;
```

Comment:
Same as before, do we need quantities to be aware of quantity_points?

Reply:
See above - I believe we need some way to easily construct quantity points from quantities, but that doesn't have to be a method. In fact `q + mean_see_level` could be the way to go!